torch/utils/benchmark/examples/blas_compare_setup.py

```diff
@@ -147,6 +148,7 @@ def main():
                 f"conda env config vars set {' '.join(env_spec.environment_variables)}",
                 shell=True,
                 capture_output=True,
+                check=True,
```

Comment:
Same as above, return code is checked on the next line
```suggestion
                check=False,
```